src/components/Card/Card.tsx

```diff
@@ -0,0 +1,248 @@
+            height,
+            icon,
+            name,
+            size = CardSize.Medium,
+            style,
+            subHeaderProps,
+            subHeaderSeparatorIcon,
+            type = CardType.list,
+            width,
+            ...rest
+        },
+        ref: Ref<HTMLDivElement>
+    ) => {
+        const largeScreenActive: boolean = useMatchMedia(Breakpoints.Large);
+        const mediumScreenActive: boolean = useMatchMedia(Breakpoints.Medium);
+        const smallScreenActive: boolean = useMatchMedia(Breakpoints.Small);
+        const xSmallScreenActive: boolean = useMatchMedia(Breakpoints.XSmall);
+
+        const contextuallySized: Size = useContext(SizeContext);
+        const mergedSize = configContextProps.noSizeContext
+            ? size
+            : contextuallySized || size;
+
+        const contextuallyDisabled: Disabled = useContext(DisabledContext);
+        const mergedDisabled: boolean = configContextProps.noDisabledContext
+            ? disabled
+            : contextuallyDisabled || disabled;
+
+        const cardClasses: string = mergeClasses([
+            styles.card,
+            {
+                [styles.cardSmall]:
+                    mergedSize === CardSize.Flex && largeScreenActive,
+            },
+            {
+                [styles.cardMedium]:
+                    mergedSize === CardSize.Flex && mediumScreenActive,
+            },
+            {
+                [styles.cardMedium]:
+                    mergedSize === CardSize.Flex && smallScreenActive,
+            },
+            {
+                [styles.cardLarge]:
+                    mergedSize === CardSize.Flex && xSmallScreenActive,
+            },
+            { [styles.list]: type === CardType.list },
+            { [styles.disabled]: allowDisabledFocus || mergedDisabled },
+            { [styles.cardLarge]: mergedSize === CardSize.Large },
+            { [styles.cardMedium]: mergedSize === CardSize.Medium },
+            { [styles.cardSmall]: mergedSize === CardSize.Small },
+        ]);
+
+        const headerClasses: string = mergeClasses([
+            styles.header,
+            headerClassNames,
+        ]);
+
+        const bodyClasses: string = mergeClasses([styles.body, bodyClassNames]);
+
+        const footerClasses: string = mergeClasses([
+            styles.footer,
+            footerClassNames,
+        ]);
+
+        return (
+            <div
+                {...rest}
+                className={cardClasses}
+                ref={ref}
+                style={{ ...style, ...{ height, width } }}
+            >
+                {children ? (
+                    children
+                ) : (
+                    <>
+                        <div className={headerClasses}>
+                            {header && header}
+                            {(icon || headerTitle) && (
+                                <div className={styles.mainHeader}>
+                                    {icon && (
```

Comment:
This should be `Icon` or `Avatar`,. use `Avatar` in Storybook. The size of both should be `40px` x `40px`